addons/mail/static/src/models/call_view.js

```diff
@@ -11,7 +11,7 @@ import { isEventHandled, markEventHandled } from '@mail/utils/utils';
 
 registerModel({
     name: 'CallView',
-    identifyingFields: ['threadView'],
+    identifyingFields: ['threadView', 'thread'],
```

Comment:
I don't think this change is correct.
Before change the modelling was a single `CallView` per `ThreadView`.
Here it looks like you can have as many `CallView` as there are `ThreadView` and `Thread`, suggesting multiplicity higher than 1 with several `Thread`, but this doesn't make sense because `ThreadView` is on a single `Thread`.